app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -241,17 +392,44 @@ export default function Navigation() {
             </div>
           </Hidden>
         </div>
-        <div className={classes.bug}>
-          <Hidden smDown>
-            <ExternalNavButton
-              route={handbookURL}
-              label={HELP}
-              labelVariant="h3"
-            />
-            <NavButton route={logoutRoute} label={LOG_OUT} />
-          </Hidden>
-          <ReportButton />
-        </div>
+
+        <Hidden smDown>
+          <div className={classes.menuContainer} ref={menuRef}>
+            <ReportButton />
+            <Button
+              className={classes.menuBtn}
+              onClick={() =>
+                setMenuOpen((prevMenuOpen: boolean) => !prevMenuOpen)
+              }
+            >
```

Comment:
```suggestion
            <Button
              aria-controls="navigation-menu"
              aria-haspopup="true"
              className={classes.menuBtn}
              onClick={() =>
                setMenuOpen((prevMenuOpen: boolean) => !prevMenuOpen)
              }
              ref={menuRef}
            >
```
As shown in the [MUI Menu Docs](https://v4.mui.com/components/menus/#menus). It's good practice for accessibility reasons again. Because of that, the `menuRef` should be referencing the button that controls whether the menu is opened or not, rather than the generic `div` container that has both this button and the report button.

Reply:
Thanks for this, i made some research because i struggled finding is MUI's doc about aria-controls for menu but found a link from their docs that redirect to w3.org that made me understand the aria-controls https://www.w3.org/TR/wai-aria-practices/#menubutton ;)